app/src/plugins/getConfig.ts

```diff
@@ -6,17 +6,14 @@ export async function getConfig (): Promise<EnvConfigI> {
   // get config from environment
   const windowLocationOrigin = window.location.origin
 
-  if (!{ appBaseURL } || !windowLocationOrigin) {
```

Comment:
Fixes this:

<img width="561" height="148" alt="image" src="https://github.com/user-attachments/assets/02d5440f-937e-4dd7-9b7e-ae0c6548393b" />

The new code should be correct if the intent is check whether the string is falsy (empty/null).

<img width="527" height="120" alt="image" src="https://github.com/user-attachments/assets/5157fbff-6a4e-4525-bca1-c312aaf5c83b" />